cmd/influxd/server_integration_test.go

```diff
@@ -1458,9 +1458,10 @@ func Test3NodeServerFailover(t *testing.T) {
 }
 
 // ensure that all queries work if there are more nodes in a cluster than the replication factor
-func Test3NodeClusterPartiallyReplicated(t *testing.T) {
+// and there is more than 1 shards
+func Test6NodeClusterPartiallyReplicated(t *testing.T) {
 	t.Parallel()
-	testName := "3-node server integration partial replication"
+	testName := "6-node server integration partial replication"
```

Comment:
Yes, a node size of 5 and a replication factor of 2 would be better. That should result in shard groups of 2 shards in size, which will then result in 4 actual shards on the cluster.